### ISSUE TITLE
Please list orelib under the list of dependencies as a QOL improvement

### ISSUE THREAD

**Issue by xkforce** [OreCruncher/DynamicSurroundings#413]
*Mod Version:*

*Forge Version:*

*Link to crash log (if applicable):*

*Description:*
The newer versions of dynamic surroundings require orelib 3.5+ as a dependency and will not run if it is missing but this mod is not listed as a dependency under the relations tab and while it is mentioned in the release notes, it would be a lot nicer if it were listed as a required dependency under the relations tab as well.

**Comment by OreCruncher** [comment]
CurseForge has it as a Required Dependency.

**Comment by xkforce** [comment]
This issue isn't really for me since I went into the release notes and actually read them but for other people that use the dependency listing in the relations menu instead of going into the release notes.  The interesting thing about it is that if you go into the relations menu on the orelib mod, dynamic surroundings is listed as a mod that requires orelib.  So there may be something not right on twitch's side or something idk.

**Comment by OreCruncher** [comment]
Yep.  The other main question is the "relations" link on the main page is tied to *which* version of the mod?  There are versions for 1.10, 1.11, 2 for 1.12, and 1.7.10.  Potluck!
